csfieldguide/static/interactives/matrix-simplifier/js/matrix-simplifier.js

```diff
@@ -51,13 +52,32 @@ m1 = mathjs.matrix([
 m2 = mathjs.matrix([[10,0,0],[0,10,0],[0,0,10]]);
 v1 = mathjs.matrix([[10], [0], [0]]);
 
-var matrices = [m1, m2];
-var vectors = [v1];
-
 // Arrays that will keep track of the order the matrices and vectors are in
 var currentMatricesOrder = [m1, m2];
 var currentVectorsOrder = [v1];
 
+m1String = [
+  ['cos(45)', '0', 'sin(45)'],
+  ['0', '1', '0'],
+  ['-sin(45)', '0', 'cos(45)']
+];
+m2String = [
+  ['10', '0', '0'],
+  ['0', '10', '0'],
+  ['0', '0', '10']
+]
+v1String = [
+  ['10'],
+  ['0'],
+  ['0']
+]
+// Arrays that will hold the string version of matrices and vectors.
+// Used for populating modal when editing a matrix or vector.
+// Needed because we don't want to display evaluated trig functions.
+// E.g we want to show cos(45) and not 0.71
+var matricesStringFormat = [m1String, m2String];
+var vectorsStringFormat = [v1String];
+
```

Comment:
(Nowhere in particular)
If there are no transform matrices the result should probably be the identity rather than 0
![image](https://user-images.githubusercontent.com/25916475/66617853-4cf32080-ec33-11e9-97fb-a9b1b37a76ab.png)
You can do this by starting with the identity matrix and multiplying each one in the list with that